### PR TITLE
Add all extension images

### PR DESCRIPTION
**Pull request by Dy-tech**
@GarboMuffin Here are the images COMPLETED.

### CODE REVIEW

**Comment by LukeManiaStudioss** [comment]
Extension thumbnails need to be 600x300 SVGs. Furthermore it's preferred that you don't put the name of the extension in the thumbnail, since you only need to look an inch down to see it.

File: images/rixxyx.svg
Comment (GrahamSH-LLK) [inline images/rixxyx.svg:1]: idk what this is but it's ugly

Comment (LukeManiaStudioss) [reply images/rixxyx.svg:1]: It's meant to be, that's the joke

File: website/index.ejs [hunk 1]
```diff
@@ -573,7 +573,7 @@
         </div>
 
         <div class="extension">
-          <%- banner('Longboost/color_channels') %>
+          <%- banner('color_channels') %>
```
Comment (GarboMuffin) [inline website/index.ejs:576]: Doing this breaks the links on the website, by the way

**Comment by GarboMuffin** [comment]
I will do this on my own

**Pull request closed without merging**